_includes/charts/tigera-operator/templates/tigera-operator/01-configmap-cluster-endpoint.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.apiHost -}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kubernetes-services-endpoint
+  namespace: tigera-operator
+data:
+  KUBERNETES_SERVICE_HOST: {{.Values.apiHost}}
+  KUBERNETES_SERVICE_PORT: "443"
```

Comment:
I'm not sure that this is 443 on all cluster.  I've certainly seen 6443.  So I think this needs to be customizable too.